addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5247,7 +5247,7 @@ msgstr ""
 #: xbmc\video\dialogs\GUIDialogAudioSubtitleSettings.cpp
 #: xbmc\video\dialogs\GUIDialogVideoSettings.cpp
 msgctxt "#12376"
-msgid "Set as default for all videos"
+msgid "Set as default for all files"
```

Comment:
> Set as default for all media sounds like a reasonable compromise.

Yes, indeed, lets do that here and now then.

What da-anda is saying especially about 

> maybe a Yes-No dialog ala "Do you want to apply these settings as new defaults for any media played?"

Thats would sound reasonable due to the scope of this setting (and the 50 page disclaimer this setting implies) but anything else at this very time for this PR except @Jalle19 compromise string suggestion would have to be for a later time by someone who can implement that.
Im concerned here about the immediate messages which I can do something about and that need fixing and leave this in a better usable/understandable position.

All other pending stuff needs to be handled better and properly of which is a tall order especially re-designs and fixing the issues they leave behind in time for any "stable" release.